html.go

```diff
@@ -123,6 +130,10 @@ func HtmlRendererWithParameters(flags int, title string,
 		renderParameters.FootnoteReturnLinkContents = `<sup>[return]</sup>`
 	}
 
+	if renderParameters.SanitizedAnchorNameOverride == nil {
+		renderParameters.SanitizedAnchorNameOverride = markdownutils.CreateGitLabAnchor
```

Comment:
Perhaps you misunderstood. I meant that `blackfriday` should continue to use `sanitized_anchor_name.Create` as the default anchor sanitizer. This PR should not change the default to `markdownutils.CreateGitLabAnchor`, that should be done in your own code that imports and uses `blackfriday`.